students/travis_nelson/lesson07/html_render.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+
+    tag = "html"
+    indent = ""
+
+    def __init__(self, content=None, **element_attribute):
+
+        if content is not None:
+            self.contents = [content]
+        else:
+            self.contents = [""]
+        if element_attribute:
+            self.attributes = element_attribute
```

Comment:
I don't think you want the `if` statement - otherwise you won't have `self.attributes` defined. By default if no kwargs are supplied it will be `{}`